examples/run_glue.py

```diff
@@ -529,6 +549,10 @@ def main():
         # Good practice: save your training arguments together with the trained model
         torch.save(args, os.path.join(args.output_dir, "training_args.bin"))
 
+        # Upload trained model and relevant files (no checkpoints)
+        if args.log_on_wandb and args.upload_model and _WANDB_AVAILABLE:
+            wandb.save(os.path.join(args.output_dir, "*.*"))
```

Comment:
Are there any other files besides the model and training_args in the output dir?

Reply:
In theory no (except for a small sample output file) but the more robust way will be to get file names from the save functions (will require to update `save_pretrained` function to return file paths used for model and config files)